lib/dom-to-react.js

```diff
@@ -26,6 +27,11 @@ function domToReact(nodes, options) {
   var node;
   var isWhitespace;
   var hasReplace = typeof options.replace === 'function';
+  var transform =
+    options.transform ||
+    function (reactNode) {
+      return reactNode;
+    };
```

Comment:
Minor: can we create a `noop` function in `utilities` as a performance optimization?

Reply:
Sure, however a noop would be `function noop(arg) {}` in my opinion, is `function returnFirstArg(arg) { return arg }` also fine?

Reply:
Yes perfect